### ISSUE TITLE
Share code via permalink

### ISSUE THREAD

**Issue by slaviqueue** [electricSemyon/browser-ide#2]
It would be zaebis if one could share current code via permalink, like:
```
dirtycode.com?code=console.log('опа, а чё это мы не энкодим?')
```

**Comment by slaviqueue** [comment]
implemented in df302a1990fd139fc9a9078f59916741ffd320e1

**Comment by slaviqueue** [comment]
Actually may be closed. One is now able to copy an url with current code.
